receipt_layoutlm/receipt_layoutlm/trainer.py

```
@@ -274,19 +289,86 @@ def _preprocess(example, tokenizer, label2id, max_len: int):
+            encoding["pixel_values"] = pixel_values
+
+            if "token_type_ids" in encoding:
+                del encoding["token_type_ids"]
+
+            return encoding
+
+        if self.training_config.model_version == "v3":
+            image_cache_dir = (
+                split_metadata.image_cache_dir
+                if split_metadata and split_metadata.image_cache_dir
+                else "/tmp/receipt_images"
```

Ruff (0.15.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/receipt_images"

(S108)